app/controllers/katello/api/v2/content_export_incrementals_controller.rb

```diff
@@ -0,0 +1,99 @@
+    param :from_history_id, :number, :desc => N_("Export history identifier used for incremental export. "\
+                                                 "If not provided the most recent export history will be used."), :required => false
+    def version
+      tasks = async_task(::Actions::Pulp3::Orchestration::ContentViewVersion::Export,
+                          content_view_version: @version,
+                          destination_server: params[:destination_server],
+                          chunk_size: params[:chunk_size_mb],
+                          from_history: @history)
+
+      respond_for_async :resource => tasks
+    end
+
+    api :POST, "/content_export_incrementals/library", N_("Performs an incremental-export of the repositories in library.  Relevant only for Pulp 3 repositories")
+    param :organization_id, :number, :desc => N_("Organization identifier"), :required => true
+    param :destination_server, String, :desc => N_("Destination Server name, required for Pulp3"), :required => false
+    param :chunk_size_mb, :number, :desc => N_("Chunk export-tarfile into pieces of chunk_size mega bytes."), :required => false
+    param :from_history_id, :number, :desc => N_("Export history identifier used for incremental export. "\
+                                                 "If not provided the most recent export history will be used."), :required => false
+    def library
+      tasks = async_task(::Actions::Pulp3::Orchestration::ContentViewVersion::ExportLibrary,
+                          @organization,
+                          destination_server: params[:destination_server],
+                          chunk_size: params[:chunk_size_mb],
+                          from_history: @history)
+      respond_for_async :resource => tasks
+    end
+
+    private
+
+    def find_exportable_content_view_version
+      @version = ContentViewVersion.exportable.find_by_id(params[:id])
+      throw_resource_not_found(name: 'content view version', id: params[:id]) if @version.blank?
+      @view = @version.content_view
+    end
+
+    def find_library_export_view
+      @view = ::Katello::ContentView.find_library_export_view(destination_server: params[:destination_server],
+                                                                organization: @organization,
+                                                                create_by_default: false)
+      if @view.blank?
+        msg = _("Unable to incrementally export. Do a Full Export the library content "\
+                "before updating from the latest increment.")
+        fail HttpErrors::BadRequest, msg
+      end
+    end
+
+    def find_history
+      if !params[:from_history_id].blank?
+        @history = ::Katello::ContentViewVersionExportHistory.find(params[:from_history_id])
+        if @history.blank?
+          throw_resource_not_found(name: 'content view version export history',
```

Comment:
Isn't 'content view version export history' kind of a misnomer now? More clear to just say 'export history'